x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/setup_guide/setup_guide.test.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { SetEnterpriseSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import { SetupGuide as SetupGuideLayout } from '../../../shared/setup_guide';
```

Comment:
I feel we should just rename `shared/setup_guide` to `shared/setup_guide_layout.tsx` and export `SetupGuideLayout`. It's confusing having a file named one way but used another way everywhere.

Reply:
Copying Constance's pattern from other components. If we want to change IMO that is beyond the scope of this PR and we should discuss as a team and change all in a separate PR

Reply:
This one's on me again, and almost definitely an alias we don't need 🤔 Historical context for this shenanigans:

1. The (very early) MVP only had a single `<SetupGuide />` component
2. When WS was added, I refactored out the instructions part of the Setup Guide to a reusable component, and renamed it `SetupGuideLayout` (because that felt more accurate?) and didn't rename the file/folder name, but now I'm thinking that was probably a garbage decision
3. Here we are today and I look like a fool

I'm thinking maybe we should just drop the `Layout` totally and leave it as `<SetupGuide />` - content-wise it's a bit more than a layout because it contains all the instructions and troubleshooting steps. It's just the marketing promo that's custom per-guide. Let me know if that sits right with you @JasonStoltz!

Reply:
^ Sorry, to clarify - in a separate PR of course